src/workspaces/dashboard/RightBoxSection.ts

```diff
@@ -32,12 +38,14 @@ export const RightBoxSection = (props: RightBoxSectionProps): ReactNode => {
           titleFirst: true,
           afterTitle,
           onOpenChanged: (panelOpen) => {
-            setPanelOpen(panelOpen);
-            Ajax().Metrics.captureEvent(Events.workspaceDashboardToggleSection, {
-              title,
-              opened: panelOpen,
-              ...extractWorkspaceDetails(workspace),
-            });
+            if (workspace) {
+              setPanelOpen(panelOpen);
```

Comment:
I don't understand why we would only want to set the panel open if there is a workspace.

Reply:
Hm, this might be an artifact from older code. I agree we always want to set the panel - workspace or not.